.github/workflows/unit_tests.yml

```diff
@@ -45,19 +51,20 @@ jobs:
       - name: Run unit tests
         run: ./gradlew test coverage
 
-      - name: Save PR number to file
-        run: echo ${{ github.event.number }} > PR_NUMBER.txt
-
-      - name: Archive PR number
-        uses: actions/upload-artifact@v4
-        with:
-          name: pr_number
-          path: PR_NUMBER.txt
-
-      - name: Archive reports
-        uses: actions/upload-artifact@v4
-        with:
-          name: unit_test_reports
-          path: |
-            **/build/**/jacoco*.xml
-            **/build/classes/
+      - if: ${{ success() }} && github.event.pull_request.head.repo.fork == false
+        name: Analyze
```

Comment:
Doesn't the `name` field need to be on the first line, and the `if` field under it (unsure if this is important or not, but at the very least, it would be more consistent/readable)?